### PR TITLE
Feature/RB-24

### PR DESCRIPTION
**Pull request by liuba83**
## Description
Create book page with description and button to buy it.

## Changes
New page for book created

## Related issues
RB-24

### CODE REVIEW

**Review by in43sh — APPROVED** [review]
Good job! Everything looks good to me.

File: src/pages/books/Book.jsx [hunk 1]
```diff
@@ -1,5 +1,124 @@
...
+        style={{ gridArea: 'image' }}
+        alt={`${title}`}
+        src={coverImageUrl}
+      />
+      <div
+        className="my-2 flex h-max w-full max-w-[18rem] flex-col items-center rounded-md border border-gray px-8 py-5"
+        style={{ gridArea: 'box' }}
+      >
+        <h1 className="col-span-full text-center font-headings text-2xl font-bold md:text-left">
+          ${price}
```
Comment (EvgeniiRychkov) [inline src/pages/books/Book.jsx]: Maybe format with cents
price.toFixed(2)

File: src/pages/books/Book.jsx [hunk 2]
```diff
@@ -1,5 +1,124 @@
...
+        )}
+      </div>
+      <div className="mb-6" style={{ gridArea: 'table' }}>
+        <table className="border-separate border-spacing-1.5">
+          <tbody>
+            {[
+              ['Condition', condition],
+              ['Author', author],
+              ['Publisher', publisher],
+              ['Published year', publishedYear],
```
Comment (EvgeniiRychkov) [inline src/pages/books/Book.jsx]: Could you delete 'Published year and put it in brackets like in Figma'
Screenshot 2024-12-27 at 8 34 01 PM

File: src/pages/books/Book.jsx [hunk 3]
```diff
@@ -1,5 +1,124 @@
...
+    return <p>Loading...</p>;
+  }
+
+  return (
+    <div className="mx-5 mt-3 grid justify-items-center gap-4 [grid-template-areas:'header''image''box''table'] md:mt-20 md:grid-cols-[1fr_1fr_1fr] md:grid-rows-[min-content_1fr] md:gap-y-0 md:[grid-template-areas:'image_header_box''image_table_box']">
+      <h1
+        className="w-full font-headings text-2xl font-bold"
+        style={{ gridArea: 'header' }}
+      >
+        {title}
```
Comment (EvgeniiRychkov) [inline src/pages/books/Book.jsx:64]: Align it to the center horizontally.

File: src/pages/books/Book.jsx [hunk 4]
```diff
@@ -1,5 +1,124 @@
...
+      >
+        <h1 className="col-span-full text-center font-headings text-2xl font-bold md:text-left">
+          ${price}
+        </h1>
+        {isLoggedIn && (
+          <>
+            <Button
+              as="button"
+              type="button"
+              className="mb-2 mt-4 w-full rounded-md bg-red px-6 py-2 font-body text-xl font-semibold tracking-wide text-white transition-transform duration-150 hover:bg-darkGreenHover active:scale-95"
```
Comment (EvgeniiRychkov) [inline src/pages/books/Book.jsx]: Change hover:bg- to be like in the other buttons with bg-red

**Comment by EvgeniiRychkov** [comment]
Good job Liuba, I just left a few comments. 
Could you add Link to BookCard on Product List Page.
And don't forget to use Lint before the PR.

**Review by EvgeniiRychkov — APPROVED** [review]

**Pull request merged**